plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -1752,7 +1753,7 @@ public static function get_subscription_list_data() {
 				'key_type'          => '',
 				'key_type_label'    => '',
 				'lifetime'          => false,
-				'product_status'    => 'publish',
+				'product_status'    => ProductStatus::PUBLISH,
```

Comment:
Is it appropriate to use `ProductStatus::PUBLISH` here? I think this is concerned with products from our marketplace, rather than products in the sense of local post objects.

Reply:
Same concern here.